src/atomate2/vasp/sets/base.py

```diff
@@ -422,10 +422,19 @@ def get_input_set(  # type: ignore
             bandgap=bandgap,
             ispin=ispin,
         )
+        site_properties = structure.site_properties
+        poscar = Poscar(
+            structure,
+            velocities=site_properties.get("velocities"),
```

Comment:
Will these guarantee the continuation of the MD?

Reply:
Actually I have discovered that if an NpT simulation is performed also the lattice velocities are added to the CONTCAR (see https://www.vasp.at/wiki/index.php/POSCAR) and should be preserved. At the moment they are completely ignored from the pymatgen Poscar object. I will add them.